backend/fms_core/tests/test_services/experiment_run_test.py

```diff
@@ -0,0 +1,144 @@
+from django.test import TestCase
+
+from fms_core.models import (
+    RunType,
+    Container,
+    Instrument,
+    PropertyType,
+    InstrumentType,
+    SampleKind,
+    ProcessMeasurement
+)
+
+from fms_core.tests.constants import create_container
+from fms_core.services import experiment_run, sample
+from datetime import datetime
+
+
+class ExperimentRunServicesTestCase(TestCase):
+    def setUp(self) -> None:
+        # Create objects
+        self.experiment_name = "TestName"
+        self.start_date = datetime.strptime("2021-01-01", "%Y-%m-%d").date()
+        self.run_type_name = "DNBSEQ"
+        self.run_type, _ = RunType.objects.get_or_create(name=self.run_type_name)
+
+        self.container, _ = Container.objects.get_or_create(
+            **create_container(name="FlowcellIllumina", barcode="FlowcellIllumina",
+                               kind="dnbseq-g400 flowcell"))
```

Comment:
This is a trick container that is named after illumina but is actually a DNBSEQ ? Might be well to name it more appropriately.